Mathlib/Data/Set/Prod.lean

```diff
@@ -21,6 +21,11 @@ This file contains basic results on the following notions, which are defined in
 * `Set.diagonal`: Diagonal of a type. `Set.diagonal α = {(x, x) | x : α}`.
 * `Set.offDiag`: Off-diagonal. `s ×ˢ s` without the diagonal.
 * `Set.pi`: Arbitrary product of sets.
+
+## Note
+
+Some theorems on `Set.pi` are misnamed. Since `Set.pi` does not have infix notation, we should
+preserve the order of appearance when naming declarations.
```

Comment:
I don't agree with this, and would suggest:
* Matching the convention used in the current file for this PR, without making any note
* Starting a zulip thread to ask which convention applies here. Note that right now the `univ_pi` is 3x more prevalent than `pi_univ`.